notify/opsgenie/opsgenie.go

```diff
@@ -120,9 +120,18 @@ func (n *Notifier) createRequest(ctx context.Context, as ...*types.Alert) (*http
 
 	tmpl := notify.TmplText(n.tmpl, data, &err)
 
-	details := make(map[string]string, len(n.conf.Details))
-	for k, v := range n.conf.Details {
-		details[k] = tmpl(v)
+	var details map[string]string
```

Comment:
Let's make map in both cases. It's not strictly needed to know size upfront -> it's only for micro optimization which is not needed here. (: